python/doc/usecases/use_case_flood_model.rst

```diff
@@ -75,20 +75,72 @@ The following figure presents the model with more details.
     :alt: flooding section details
     :width: 50%
 
-    Flooding section detail
-
-If we substitute the parameters into the equation, we get:
-
- .. math::
-    S = \left(\frac{Q}{300 K_s \sqrt{\frac{Z_m - Z_v}{5000}}}\right)^{3/5} + Z_v - 58.5.
-
-
-We assume that the four inputs have the following distributions:
-
-  - :math:`Q` ~ Gumbel(mode=1013, scale=558), :math:`Q` > 0
-  - :math:`K_s` ~ Normal(mu=30.0, sigma=7.5), :math:`K_s` > 0
-  - :math:`Z_v` ~ Uniform(a=49, b=51)
-  - :math:`Z_m` ~ Uniform(a=54, b=56)
+    **Figure 2.** Flooding section detail.
+
+The cost :math:`C` can be decomposed into the building of the dyke and
+the cost of the flood ([iooss2015]_ eq. 5.3 page 103):
+
+.. math::
+    C = C_d + C_s
+
+where the cost of the dyke is:
+
+.. math::
+    C_d
+    =\begin{cases}
+    \frac{8}{20}   & \textrm{if } H_d < 8 \\
+    \frac{H_d}{20} & \textrm{otherwise},
+    \end{cases}
+
+and the cost of the flood is:
+
+.. math::
+    C_s
+    =\begin{cases}
+    1 - 0.8 \exp(-\frac{1000}{S^4}) & \textrm{if } S < 0, \\
+    1 & \textrm{otherwise}.
+    \end{cases}
+
+
+We assume that the 8 inputs have the following distributions.
+We consider 2 different set of hypotheses.
+
+- In the hypothesis where :math:`H_d` is low, then its distribution
+  is uniform in the interval :math:`[2, 4]`.
+- In the hypothesis where :math:`H_d` is high, then its distribution
+  is uniform in the interval :math:`[7, 9]`.
+  This is the hypothesis used in [iooss2015]_.
+
++----------------+-----------------------------------------------+
+| Input variable | Distribution                                  |
++================+===============================================+
+| :math:`Q`      | Gumbel(mode=1013, scale=558), :math:`Q` > 0   |
++----------------+-----------------------------------------------+
+| :math:`K_s`    | Normal(mu=30.0, sigma=7.5), :math:`K_s` > 0   |
++----------------+-----------------------------------------------+
+| :math:`Z_v`    | Uniform(a=49, b=51)                           |
++----------------+-----------------------------------------------+
+| :math:`Z_m`    | Uniform(a=54, b=56)                           |
++----------------+-----------------------------------------------+
+| :math:`B`      | Triangular(min=295, mode=300, max=305)        |
++----------------+-----------------------------------------------+
+| :math:`L`      | Triangular(min=4990, mode=5000, max=5010)     |
++----------------+-----------------------------------------------+
+| :math:`Z_b`    | Triangular(min=55, mode=55.5, max=56)         |
++----------------+-----------------------------------------------+
+| :math:`H_d`    | Uniform(min=2, max=4)                         |
++----------------+-----------------------------------------------+
+
+**Table 1.** Distribution of the input random variables in the scenario where :math:`H_d` is low.
```

Comment:
By convention, legend of Tables could be set at the top of the corresponding tables

Reply:
I am not against this convention, but this would change several other examples, including [this one](https://github.com/openturns/openturns/blob/31fc1f66f383304a354eafd6bb674ce0dc1c9260/python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_quickstart.py#L132), [that one](https://github.com/openturns/openturns/blob/31fc1f66f383304a354eafd6bb674ce0dc1c9260/python/doc/theory/reliability_sensitivity/sensitivity_sobol.rst#L365), [this one](https://github.com/openturns/openturns/blob/31fc1f66f383304a354eafd6bb674ce0dc1c9260/python/src/FunctionalChaosSobolIndices_doc.i.in#L44), and [this one too](https://github.com/openturns/openturns/blob/31fc1f66f383304a354eafd6bb674ce0dc1c9260/python/doc/theory/data_analysis/kernel_smoothing.rst#L196). Would you like that I change other examples too?